Dockerfile

```diff
@@ -2,14 +2,12 @@
 # STAGE 1.1: builder frontend
 ###################
 
-FROM node:12.20.1-alpine as frontend
+FROM metabase/ci:latest as frontend
```

Comment:
`latest` is rather dangerous.
Is there a specific, immutable tag that can be used instead?

Reply:
yes, absolutely. But in this case we control the 'latest' tag in the https://github.com/metabase/metabase-docker-ci repo. I left it that way so we can update the builder image in the other repo (if there is a Lein or Clojure update) and this one will be automatically updated as well

Reply:
Just built and image and pinned down the tag